.drone.yml

```
@@ -1,83 +1,111 @@
 ---
 kind: pipeline
 type: kubernetes
-name: deploy
+name: promote
 trigger:
   event:
     - promote
 
+environment:
+  CHART_NAME: hocs-templates
+  KUBE_CLUSTER: acp-notprod
+  KUBE_SERVER: https://kube-api-notprod.notprod.acp.homeoffice.gov.uk
+
 steps:
-  - name: fetch and checkout
+  - name: checkout
     image: alpine/git
     commands:
       - git fetch --tags
-      - git checkout $${VERSION}
+      - git checkout ${VERSION}
+      - git submodule update --init --recursive
 
-  - name: deploy to cs-qa
-    image: quay.io/ukhomeofficedigital/kd
-    commands:
-      - cd kube
-      - ./deploy.sh
+  - name: cs-qa
+    image: quay.io/ukhomeofficedigital/hocs-helm-kubectl:1.0.0
     environment:
-      ENVIRONMENT: cs-qa
+      KUBE_NAMESPACE: cs-qa
       KUBE_TOKEN:
-        from_secret: hocs_templates_cs_qa
-      KUBE_SERVER: https://kube-api-notprod.notprod.acp.homeoffice.gov.uk
+        from_secret: cs_notprod
+    commands:
+      - ./ci/helm/helm-deploy.sh --values ./helm/values-notprod.yaml --values ./helm/values-qa.yaml
+    depends_on:
+      - checkout
     when:
       target:
-        - release
-    depends_on:
-      - fetch and checkout
+        - cs-qa
```

Weird indentation here.
```suggestion
      - cs-qa
```

---

I think this has been corrected now, I can't find it now.